### ISSUE TITLE
measuring distance icon and shortcut

### ISSUE THREAD

**Issue by mjeffe98** [ngageoint/hootenanny-ui#26]
@sisskind would it be possible to change the hand icon when measuring distance to something like a ruler or an icon to distinguish itself from the standard pan/drag?  It would be nice to know that users are in measure as opposed to active map mode where they can resume panning.  Also it might not be a bad idea to include a shortcut key so you don't have to click on the button to instantiate the tool.

**Comment by sisskind** [comment]
@mjeffe98 can do both.  will rename to reflect icon/shortcut.

**Comment by sisskind** [comment]
@mjeffe98 new icon ready to test in develop.  Shortcut will be worked on after #9 is merged in.

**Comment by sisskind** [comment]
Added in commit (https://github.com/ngageoint/hootenanny-ui/commit/9fa8d3213cefae197e06334c72d4a3fe4dd9fc58)

Shortcuts:
* Measure Add Line: 6
* Measure Add Area: 7
* Clip Bounding Box: 8
* Clip Visual Extent: 9

Will add in tool tip to reflect this and then merge into develop for testing.

**Comment by sisskind** [comment]
@mjeffe98 Ready for testing.

**Comment by sisskind** [comment]
https://github.com/ngageoint/hootenanny-ui/pull/54
